sdk/resourcemanager/azure-resourcemanager/src/samples/java/com/azure/resourcemanager/ThreadPoolSamples.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import reactor.netty.resources.ConnectionProvider;
+import reactor.netty.resources.LoopResources;
+
+import java.time.Duration;
+import java.util.concurrent.ForkJoinPool;
+
+/**
+ * Code samples for the THREAD_POOL_AND_CONNECTION_POOL.md
+ */
+public class ThreadPoolSamples {
+
+    public AzureResourceManager azureClientThreadPool(String subscriptionId) {
+        NettyAsyncHttpClientBuilder singletonHttpClientBuilder = new NettyAsyncHttpClientBuilder();
+        // BEGIN: readme-sample-azureClientConnectionPool
```

Comment:
Should mention somewhere, either in the code samples or in the README that uses these code samples, that this example is only for configuring the Reactor Netty HttpClient implementation as there are configurations available for the OkHttp HttpClient implementation as well.

Reply:
Added reference to both `azure-core-http-netty` and `azure-core-okhttp`.